src/components/Login/BlockUser.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FunctionComponent, useEffect } from "react";
+import { View, StyleSheet, Image } from "react-native";
+import { InitialisationContainerNavigationProps } from "../../types";
+import { size } from "../../common/styles";
+import { Credits } from "../Credits";
+import { Sentry } from "../../utils/errorTracking";
+import * as Linking from "expo-linking";
+import { DarkButton } from "../Layout/Buttons";
+import { AppText } from "../Layout/AppText";
+import { useTranslate } from "../../hooks/useTranslate/useTranslate";
+
+const styles = StyleSheet.create({
+  asset: {
+    width: 203,
+    height: 206,
+    center: true,
+    marginBottom: size(8),
+  },
+  body: {
+    textAlign: "center",
+    font: "IBM Plex Sans",
+    fontSize: 16,
+    color: "#292B2C",
+    fontWeight: "400",
+    lineHeight: 24,
+    width: 279,
+    paddingBottom: size(2),
+  },
+  centerContent: {
+    alignItems: "center",
+  },
+  credits: {
+    bottom: size(3),
+  },
+  container: {
+    justifyContent: "center",
+    backgroundColor: "white",
+    width: "100%",
+    height: "100%",
+  },
+  heading: {
+    textAlign: "center",
+    font: "IBM Plex Sans",
+    fontSize: 20,
+    color: "#292B2C",
+    fontWeight: "700",
+    lineHeight: 28,
+    width: 279,
+    paddingBottom: size(1),
+  },
+  launchWebAppButton: {
+    position: "absolute",
+    bottom: size(10),
+    width: 350,
+    alignSelf: "center",
+    marginBottom: size(2),
+  },
+});
```

Comment:
Following the design specs given in [figma](https://www.figma.com/file/UyHLjZZ8RbFkBYxa1peLQ5/Mobile-App?type=design&node-id=7145%3A107838&mode=design&t=MXblIBQxIRUR3MPw-1).

However, my margin/padding could be wrong 😅.